README.md

```diff
@@ -232,7 +232,7 @@ Note about the implementation: This is not supported by JGit, so it makes use of
 
 ## Diffs
 
-Modifications contains _diffs_ from the current version and the last one. This diff is extracted
+Modifications contains _diffs_ between two specified versions. This diff is extracted
```

Comment:
Maybe we should keep here as it was? Because in a normal execution, that's what the diff means in a `Modification`. Then, your following explanation sounds good to me.

What do you think?